### DISCUSSION TITLE
custom.setHttpOptionsDefaults 4.74 to 5.3.0

### DISCUSSION THREAD

**Discussion by xDestyn** [panva/openid-client#542] [Q&A] [answered]
Upgrading openid-client from 4.74 to 5.30.
Also, upgraded jose to the latest 4.11.0 version.
I see error: 
`Argument of type '{ rejectUnauthorized: boolean; }' is not assignable to parameter of type 'Partial<Pick<RequestOptions, "agent" | "ca" | "cert" | "crl" | "headers" | "key" | "lookup" | "passphrase" | "pfx" | "timeout">>'.
  Object literal may only specify known properties, and 'rejectUnauthorized' does not exist in type 'Partial<Pick<RequestOptions, "agent" | "ca" | "cert" | "crl" | "headers" | "key" | "lookup" | "passphrase" | "pfx" | "timeout">>'.ts(2345)`

Is rejectUnauthorized not supported anymore as far as setting in the custom.setHttpOptionsDefaults method?

**Comment by panva** [comment, marked as answer]
The available options are documented in the respective API section. https://github.com/panva/node-openid-client/tree/main/docs#customizing

**Reply by panva** [reply]
The following options can be provided agent, ca, cert, crl, headers, key, lookup, passphrase, pfx, timeout.

**Reply by xDestyn** [reply]
Thank you for clarifying.
